pkgs/tools/misc/lerpn/default.nix

```
@@ -0,0 +1,23 @@
+{ python3, lib, fetchFromGitea }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "lerpn";
+  version = "unstable-2023-06-08";
+
+  src = fetchFromGitea {
+    domain = "gitea.alexisvl.rocks";
+    owner = "alexisvl";
+    repo = pname;
+    rev = "df792ae930005415afd4e56950de671cdd23fb26";
+    sha256 = "sha256-XfQssik8POBgfCbmP3tjUGomE769yrujqJhImd6vxTQ=";
+  };
+
+  dontUseSetuptoolsCheck = true; # doesn't halt
+
+  meta = with lib; {
+    homepage = "https://gitea.alexisvl.rocks/alexisvl/lerpn";
+    description = "Curses RPN calculator written in straight Python";
+    maintainers = with maintainers; [ ckie ];
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Plus;
```
see
https://gitea.alexisvl.rocks/alexisvl/lerpn/src/commit/b65e56cfbbb38f8200e7b0c18b3a585ae768c6e2/LerpnApp/__main__.py#L5-L6

---

ah, missed that.